editor.html

```
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+!DOCTYPE html>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)